presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1172,17 +1172,17 @@ std::vector<std::string> PrestoServer::registerConnectors(
     const fs::path& configDirectoryPath) {
   static const std::string kPropertiesExtension = ".properties";
 
-  const auto numConnectorIoThreads = std::max<size_t>(
-      SystemConfig::instance()->connectorNumIoThreadsHwMultiplier() *
+  const auto numConnectorCpuThreads = std::max<size_t>(
+      SystemConfig::instance()->connectorNumCpuThreadsHwMultiplier() *
           std::thread::hardware_concurrency(),
       0);
```

Comment:
Can you guys confirm if we still need this multiplier mechanism to derive # of cpu threads?

I will double check and handle the config names later. It's being referenced and set to 0 in prod.